validation/services/user.js

```diff
@@ -0,0 +1,16 @@
+const allowedUserFieldsForUpdate = {
+  photo: true,
+  firstName: true,
+  lastName: true,
+  address: {
+    country: true,
+    city: true
+  },
+  education: true,
+  categories: true,
+  nativeLanguage: true,
+}
+
+module.exports = {
+  allowedUserFieldsForUpdate
+}
```

Comment:
ok, but use it for different endpoint

Reply:
i think that now it's useless and need to be deleted